API.md

```diff
@@ -126,3 +126,53 @@ bool provision(const byte appEui[8], const appKey[16]);
 
 - `const byte appEui[8]`: Application Identifier for the device.
 - `const byte appKey[16]`: Application Key assigned to the device.
+
+## Class: TheThingsMessage
+
+```c
+#include <TheThingsNetwork.h>
+
+TheThingsMessage msg(Stream& modemStream);
+```
+
+- `Stream& debugStream`: Stream to write debug logs to (for The Things Node/Uno use `Serial` and data rate `9600`).
```

Comment:
Baud rate

Also, where's the `debugStream` in the constructor? It only says `modemStream`